tests/Tests/500__jupyterhub/test-minimal-image.robot

```diff
@@ -35,6 +37,25 @@ Can Spawn Notebook
     Fix Spawner Status
     Spawn Notebook With Arguments    image=s2i-minimal-notebook
 
+Verify Tensorflow Can Be Installed In The Minimal Python Image Via Pip
+    [Documentation]    Verify Tensorflow Can Be Installed In The Minimal Python image via pip
+    [Tags]    ODS-555    ODS-908    ODS-535
+    ...       Tier1
+    Clone Git Repository    ${RequirementsFileRepo}
+    Open New Notebook
+    Close Other Tabs
+    Add and Run JupyterLab Code Cell In Active Notebook    !pip install -r useful-files/requirements.txt --progress-bar off
+    Wait Until JupyterLab Code Cell Is Not Active
+    ${version} =    Run Cell And Get Output
+    ...    !pip show tensorflow | grep Version: | awk '{split($0,a); print a[2]}' | awk '{split($0,b,"."); printf "%s.%s.%s", b[1], b[2], b[3]}'
+    Should Be Equal    2.7.0    ${version}
+    Add and Run JupyterLab Code Cell In Active Notebook    !pip install --upgrade tensorflow --progress-bar off
+    Wait Until JupyterLab Code Cell Is Not Active
+    ${updated version} =    Run Cell And Get Output
+    ...    !pip show tensorflow | grep Version: | awk '{split($0,a); print a[2]}' | awk '{split($0,b,"."); printf "%s.%s.%s", b[1], b[2], b[3]}'
+    Should Not Be Equal    ${updated version}    ${version}
+    Clean Up User Notebook    ${OCP_ADMIN_USER.USERNAME}    ${TEST_USER.USERNAME}
```

Comment:
You should use `Clean Up Server` instead. https://github.com/red-hat-data-services/ods-ci/blob/master/tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot#L151